src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -44,22 +44,13 @@ const HealthCare = ({
 
   useEffect(
     () => {
-      if (canAccessMessaging && !dataLoadingDisabled) {
+      if (shouldFetchMessages && !dataLoadingDisabled) {
```

Comment:
Even though we always show the Unread Messages link, I believe it makes sense to keep conditionally fetching messages. From what I understand, if the user does not have a premium MHV account, they can sign up for one, but they would have no messages in their inbox until they sign up for the service.